packages/fx-core/templates/plugins/resource/frontend/bicep/frontendHosting.bicep

```diff
@@ -1,11 +1,14 @@
-param frontend_hosting_storage_name string
+@minLength(3)
+@maxLength(24)
+@description('Name of Storage Accounts for frontend hosting.')
```

Comment:
How do we determine whether to add constraints and description to a parameter? Seems some parameters have constraints / description and some parameters do not have

Reply:
Same here. We can come out with a consistent practice for each resource plugins.